src/main/features/linux/mprisService.js

```diff
@@ -0,0 +1,60 @@
+const mpris = require('mpris-service');
+
+class MprisService {
+
+    constructor() {
+        this.player = mpris({
+            name: 'gpmdp',
+            identity: 'Google Play Music Desktop Player',
+            supportedInterfaces: ['player']
+        });
+
+        this.listeners();
+    }
+
+    listeners() {
+        this.player.on('play', () => {
+            if (!PlaybackAPI.isPlaying()) {
+                Emitter.sendToGooglePlayMusic('playback:playPause');
+            }
+        });
+
+        this.player.on('playpause', () => {
+           Emitter.sendToGooglePlayMusic('playback:playPause');
+        });
+
+        this.player.on('next', () => {
+           Emitter.sendToGooglePlayMusic('playback:nextTrack');
+        });
+
+        this.player.on('previous', () => {
+           Emitter.sendToGooglePlayMusic('playback:previousTrack');
+        });
+
+        this.player.on('stop', () => {
+           Emitter.sendToGooglePlayMusic('playback:stop');
+           this.updatePlaybackStatus('Stopped');
+        });
+
+        PlaybackAPI.on('change:song', (newSong) => {
+            this.updateMetadata(newSong);             
+        });
+    }
+
+    updateMetadata(newSong) {
+        this.player.metadata = {
+            // This is required in MPRIS, but no uuid for each track is given by gmusic library.
+            //'mpris:trackid': player.objectPath('track/0'),
```

Comment:
Honestly I'm not sure it's entirely required, after reviewing docs here:

https://www.freedesktop.org/wiki/Specifications/mpris-spec/metadata/#index1h4

It seems to be relevant when using TrackList only, which I didn't implement because gmusic doesn't implement playlist features.